packages/web-components/src/components/va-banner/va-banner.tsx

```diff
@@ -86,8 +92,14 @@ export class VaBanner {
   })
   componentLibraryAnalytics: EventEmitter;
 
-  /* eslint-disable-next-line i18next/no-literal-string */
-  private prepareBannerID = () => `${this.headline}:${this.el.innerHTML}`;
+  /**
+   * Prepares the BannerID to be used for identifying and labeling dismissed banners in
+   * DIMISSED_BANNERS localStorage. If dismissedBannerID is provided that will be returned,
+   * otherwise it will default to contents of the banner
+   */
+  private prepareBannerID = () =>
+    /* eslint-disable-next-line i18next/no-literal-string */
+    this.dismissedBannerId || `${this.headline}:${this.el.innerHTML}`;
```

Comment:
```suggestion
    (this.dismissedBannerId && this.dismissedBannerId.length) || `${this.headline}:${this.el.innerHTML}`;
```

Reply:
Because a prop can exist without specifying a value (e.g. `<va-banner dismissed-banner-id />`) it's always a good idea to check that there's a value there, otherwise the "prepared banner id" would be an empty string.

Reply:
That makes sense to me, but I believe we will need to expand this a bit to ensure it returns the dismissedBannerID vs a boolean 
```suggestion
    this.dismissedBannerId && this.dismissedBannerId.length ? this.dismissedBannerId : `${this.headline}:${this.el.innerHTML}`;
```